src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -493,11 +493,21 @@ private void GenerateFunctions(IEnumerable<Type> directTypes)
         /// </summary>
         private void InitializeFunctionDescriptors(Collection<FunctionMetadata> functionMetadata)
         {
-            _descriptorProviders = new List<FunctionDescriptorProvider>()
+            var language = _settingsManager.Configuration.GetSection("workers:config:language").Value;
```

Comment:
language read from config is used in InitializeFunctionDescriptors and InitializeWorkers method. Can you move looking up language to [Initialize()](https://github.com/christopheranderson/azure-functions-host/blob/21ce0a4363df26754dd614ae06a8e6113ce47bb8/src/WebJobs.Script/Host/ScriptHost.cs#L291)